tests/test_twitterCleanup.py

```diff
@@ -0,0 +1,26 @@
+from unittest import TestCase, mock
+import datetime
+
+import twitter_cleanup
+
+
+class TwitterCleanupTest(TestCase):
+    @mock.patch("twitter_cleanup.API")
+    @mock.patch("twitter_cleanup.Authentication")
+    def test_progress(self, mock_auth, api_mock):
+        cleaner = twitter_cleanup.TwitterCleanup()
+        progress = cleaner.progress_bar_kwargs("object", 100)
+        self.assertEqual(progress["label"], "Looking for object in 100 accounts")
+
+    @mock.patch("twitter_cleanup.API")
+    @mock.patch("twitter_cleanup.Authentication")
+    def test_action_message(self, mock_auth, api_mock):
+        cleaner = twitter_cleanup.TwitterCleanup()
+        mock_user = mock.Mock(
+            status=mock.Mock(created_at=datetime.datetime(2019, 12, 2, 0, 0, 0, 0))
+        )
+        mock_user.status.text = "Hello"
+        mock_user.screen_name = "MrUser"
```

Comment:
Is there a special reason you used the _argument_ architecture for the `.status.crated_at`, but the _attribute_ archtecture_ for other values? I mean, can we have the same strategy in mocking here?

Either:

```python
user = Mock(
    status=Mock(
        created_at=datetime(2019, 12, 2, 0, 0, 0, 0),
        text="Hello"
    ),
    screen_name="MrUser"
)
```

Or:

```python
user = Mock()
user.status.created_at = datetime(2019, 12, 2, 0, 0, 0, 0)
user.status.text = "Hello"
user.screen_name = "MrUser"
```

Personally I find the last one more readable and in line with what you suggested in `test_user.py` ; )